CMakeLists.txt

```diff
@@ -465,16 +465,19 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/SipiConfig.h.in ${CMAKE_CURRENT_SOURC
 # Here we get the ICC profiles from the Adobe website and prepare them for including
 # We just use the AdobeRGB1998.cc and the USWebCoatedSWOP.icc profiles for now...
 #
+unset(ICC CACHE)
```

Comment:
this unset is needed otherwise the variable ICC is saved in CMakeCache.txt and not updated.